frontend/www/js/omegaup/components/problem/Collection.vue

```diff
@@ -18,11 +18,21 @@
         <div class="container-fluid">
           <div class="row">
             <omegaup-problem-collection
-              v-for="(collect, idx) in problemCount"
+              v-for="(collection, idx) in problemCount"
               :key="idx"
-              :level-tag-alias="collect.name"
-              :problem-count="collect.problems_per_tag"
-            ></omegaup-problem-collection>
+              :title="getname(collection.name)"
+              :problem-count="collection.problems_per_tag"
+              :href="'/problem/collection/' + collection.name + '/'"
```

Comment:
```suggestion
              :href="`/problem/collection/${encodeURIComponent(collection.name)/`"
```

así evitamos ataques posibles de inyección de código.

Reply:
me genera un problema por la parte de las comillas, a partir de la segunda `  todo me lo marca como comentario, cómo se podría poner? o no lleva comillas?